kwasm-lemmas.md

```diff
@@ -71,6 +221,13 @@ We want to make the variant explicit, so we introduce the following helper, whic
       requires notBool isInt(I)
 ```
 
+We can now state the intended semantics of `#get`.
+
+```k
+    rule 0              <=Int #get(BM, ADDR) => true requires #isByteMap(BM) [simplification]
+    rule #get(BM, ADDR)  <Int 256            => true requires #isByteMap(BM) [simplification]
```

Comment:
Do these help? Try removing

Reply:
Can we get a log to find out what applies?
Can I automatically detect rule applications? Good for report/general grokking.